projects/ddap-common-lib/src/lib/layout/sidenav-left/sidenav-left.component.ts

```diff
@@ -30,4 +32,12 @@ export class SidenavLeftComponent {
     return true;
   }
 
+  shouldExpandPanel(group: GroupMetadata) {
+    for (const submodule of this.viewController.getGroupSubmodules(group)) {
+      if (this.router.url.includes(submodule.routerLink)) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
Even though your solution is good, you should be able to simplify it further:

```suggestion
    this.viewController.getGroupSubmodules(group).some((submodule) => this.router.url.includes(submodule.routerLink));
```